tests/python/cpu/test_mkldnn.py

```diff
@@ -41,18 +42,49 @@ def test_mkldnn_install():
         import mxnet as mx
     except (ImportError, OSError) as e:
         assert 0, "Import mxnet error: %s. Please double check your build/" \
-               "install steps or environment variable settings" % str(e)
+            "install steps or environment variable settings" % str(e)
 
     pid = os.getpid()
-    rc = os.system("cat /proc/" + str(pid) + \
-                       "/maps | grep libmkldnn > /dev/null")
+    rc = os.system("cat /proc/" + str(pid) +
+                   "/maps | grep libmkldnn > /dev/null")
 
     if rc == 0:
         logging.info("MXNet is built/installed correctly with MKL-DNN")
     else:
         assert 0, "MXNet is built/installed incorrectly with MKL-DNN, please " \
-               "double check your build/install steps or environment " \
-               "variable settings"
+            "double check your build/install steps or environment " \
+            "variable settings"
+
+
+def test_mkldnn_model():
```

Comment:
Please explain that this test has no asserts because it is only supposed to run successfully without failure.

Reply:
@marcoabreu i added assert for sanity check, and renamed file as you mentioned. any other suggestions , please let me know.

Reply:
Thanks!